conf/app.routes

```diff
@@ -116,4 +116,7 @@ GET        /group-cannot-apply        uk.gov.hmrc.plasticpackagingtax.registrati
 POST       /group-cannot-apply        uk.gov.hmrc.plasticpackagingtax.registration.controllers.NotMembersUnderGroupControlController.submit()
 
 GET        /group-member-organisation        uk.gov.hmrc.plasticpackagingtax.registration.controllers.group.OrganisationDetailsTypeController.displayPage()
-POST       /group-member-organisation        uk.gov.hmrc.plasticpackagingtax.registration.controllers.group.OrganisationDetailsTypeController.submit()
\ No newline at end of file
+POST       /group-member-organisation        uk.gov.hmrc.plasticpackagingtax.registration.controllers.group.OrganisationDetailsTypeController.submit()
+
+GET        /organisation-remove/:memberId     uk.gov.hmrc.plasticpackagingtax.registration.controllers.group.RemoveMemberController.displayPage(memberId)
+POST       /organisation-remove/:memberId     uk.gov.hmrc.plasticpackagingtax.registration.controllers.group.RemoveMemberController.submit(memberId)
```

Comment:
Could you move the `/group-organisation-list ` definitions to be near these two controllers?
I think ultimately these sets of "group-member" urls could do with name consolidation